### ISSUE TITLE
Jest/Enzyme tests failing with asField

### ISSUE THREAD

**Issue by cloudsociety** [joepuzzo/informed#164]
Hi,

I've created a date-picker that works as I'd expect it to, but I'm getting the following error when running my tests:

```
<Datepicker> › should render a datepicker input

    TypeError: I.getFullField is not a function

       6 | describe("<Datepicker>", () => {
       7 |   it("should render a datepicker input", () => {
    >  8 |     const wrapper = shallow(<Datepicker
         |                     ^
       9 |       label="Pick-up Date"
      10 |       field="datepicker"
      11 |     />);

      at useRef (node_modules/informed/dist/webpack:/informed/src/hooks/useField.js:172:15)
      at initialValue (node_modules/informed/dist/webpack:/informed/src/HOC/asField.js:25:5)
      at node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js:483:38
      at ReactShallowRenderer.render (node_modules/react-test-renderer/cjs/react-test-renderer-shallow.development.js:716:41)
      at node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js:487:37
      at withSetStateAllowed (node_modules/enzyme-adapter-utils/build/Utils.js:143:16)
      at Object.render (node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js:486:70)
      at new ShallowWrapper (node_modules/enzyme/build/ShallowWrapper.js:265:22)
      at Object.shallow (node_modules/enzyme/build/shallow.js:21:10)
      at Object.<anonymous> (src/components/Datepicker/index.test.tsx:8:21)
```

The test is pretty simple:

```javascript
import { shallow } from "enzyme";
import React from "react";

import Datepicker from "./index";

describe("<Datepicker>", () => {
  it("should render a datepicker input", () => {
    const wrapper = shallow(<Datepicker
      label="Pick-up Date"
      field="datepicker"
    />);
    expect(wrapper.find("label").html()).toMatch(/Pick-up Date/);
  });
});
```

And I'm hoping the component is too:

```javascript
import React, { useMemo, useCallback } from "react";
import DatePicker from "react-datepicker";
import classnames from "classnames";
import { asField } from "informed";

import "react-datepicker/dist/react-datepicker.css";
import styles from "../Input/index.module.scss";

interface DatepickerProps {
  label: string;
  field: string;
  initialDate?: any;
  filterDate?: any;
  dateFormat?: string;
}

const Datepicker = asField<DatepickerProps>(
  ({label, field, initialDate, fieldState, fieldApi, ...otherOptions}) => {

    const value = useMemo(() => {
      return fieldState.value || initialDate || new Date();
    }, [fieldState]);

    const memoHandleOnChange = useCallback((date: any) => {
        fieldApi.setValue(date);
      },
      [fieldApi.setValue]
    );

    return (
      <div className={classnames(styles["InputText"])}>
        <label htmlFor={field}>{label}</label>
        <DatePicker
          selected={new Date(value.toString())}
          onChange={memoHandleOnChange}
          id={field}
          name={field}
          {...otherOptions}
        />
      </div>
    );
  }
)

export default Datepicker;
```

**Not sure what I'm missing, but if anyone has some advice, I'd be grateful.**

**Comment by joepuzzo** [comment]
Ahh i think this is small bug im fixing now!!

**Comment by joepuzzo** [comment]
Fixed in V2.1.10!

**Comment by cloudsociety** [comment]
I'll give it a try. Thanks, @joepuzzo. Appreciate the quick turn-around. 🙂
